Sources/SourceGraph/SourceGraph.swift

```diff
@@ -85,6 +87,22 @@ public final class SourceGraph {
         _ = redundantPublicAccessibility.removeValue(forKey: declaration)
     }
 
+    func markRedundantInternalAccessibility(_ declaration: Declaration, file: SourceFile) {
+        redundantInternalAccessibility[declaration, default: []].insert(file)
+    }
+
+    func unmarkRedundantInternalAccessibility(_ declaration: Declaration) {
+        _ = redundantInternalAccessibility.removeValue(forKey: declaration)
+    }
```

Comment:
This is unused?

Reply:
I guess all PRs in this repo should be self-checked by periphery 😁